### DISCUSSION TITLE
[Bug] Comment shortcut not working for custom language syntax definition

### DISCUSSION THREAD

**Discussion by xleepy** [microsoft/monaco-editor#3840] [Q&A] [answered]
Hi, noticed an issue with custom language definition and shortcuts support. 
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Link

https://microsoft.github.io/monaco-editor/monarch.html

### Reproduction Steps

Try to use cmd + / (on macOS) to comment `Console.writeln( "Hello Monarch world\n")` 

### Actual (Problematic) Behavior

Nothing happens and user needs to manually type comment pattern. 

### Expected Behavior

Line will be commented out.

### Additional Context

This happens only if custom language syntax definition used

**Comment by spahnke** [comment, marked as answer]
The Monarch playground only allows you to specify the syntax highlighting part of a language that you would feed into `monaco.languages.setMonarchTokensProvider` IIRC. However, there is a second part to a language that is set by calling `monaco.languages.setLanguageConfiguration` which includes information like comment characters, bracket characters, etc. Only if you specify this second part the editor knows about comment characters and can apply the "toggle line comment" command.

**Reply by xleepy** [reply]
Would be nice to have it documented somewhere or have some guide because was a bit lost in current documentation 😅 and thanks I'll try!
